content/en/user-guide/chaos-engineering/fault-injection-service/index.md

```diff
@@ -1,50 +1,49 @@
 ---
-title: "Using the Fault Injection Service"
-linkTitle: "Using the Fault Injection Service"
-weight: 1
-description: Use LocalStack Outages Extension to mimic service outages by testing your infrastructure's ability to deploy robustly and recover from unexpected events.
+title: "Fault Injection Service"
+linkTitle: "Fault Injection Service"
+description: Use Fault Injection Service to simulate faults in your infrastructure and test its fault tolerance
 tags: ["Pro image"]
 ---
 
 ## Introduction
 
-The [AWS Fault Injection Service](https://aws.amazon.com/fis/) is a fully managed service designed to help you improve
-the resilience of your applications by simulating
-real-world outages and operational issues. This service allows you to conduct controlled experiments on your AWS
-infrastructure, injecting
-faults and observing how your system responds under various conditions. By using the Fault Injection Service, you can
-identify weaknesses,
-test recovery procedures, and ensure that your applications can withstand unexpected disruptions. This proactive
-approach to reliability
-engineering enables you to enhance system robustness, minimize downtime, and maintain a high level of service
-availability for your users.
+The [Fault Injection Service](https://aws.amazon.com/fis/) is a fully managed service designed to help you improve the resilience of your applications by simulating real-world outages and operational issues.
+This service allows you to conduct controlled experiments on your AWS infrastructure, injecting faults and observing how your system responds under various conditions.
 
-To see the FIS service in action within a more complex application stack, please refer to the [Chaos Engineering Tutorials]({{< ref "tutorials" >}}).
+By using the Fault Injection Service, you can identify weaknesses, test recovery procedures, and ensure that your applications can withstand unexpected disruptions.
+This proactive approach to reliability engineering enables you to enhance system robustness, minimize downtime, and maintain a high level of service availability for your users.
 
```

Comment:
maybe add a bit of context about our chaos features. so that people landing on this page directly, also understand where this fits in our wider chaos features.